src/main/java/seedu/address/logic/commands/EditEntryCommand.java

```diff
@@ -0,0 +1,197 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.util.CliSyntax.PREFIX_AMOUNT;
+import static seedu.address.logic.parser.util.CliSyntax.PREFIX_CATEGORY;
+import static seedu.address.logic.parser.util.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.util.CliSyntax.PREFIX_TAG;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.category.Category;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.account.ActiveAccount;
+import seedu.address.model.account.entry.Amount;
+import seedu.address.model.account.entry.Description;
+import seedu.address.model.account.entry.Entry;
+import seedu.address.model.account.entry.Expense;
+import seedu.address.model.account.entry.Revenue;
+import seedu.address.model.tag.Tag;
+
+public class EditEntryCommand extends Command {
```

Comment:
This can be called EditCommand, if not need to add Entry to AddCommand and DeleteCommand as well